go/vt/sqlparser/ast.go

```diff
@@ -1711,13 +1711,20 @@ func (node *ValuesFuncExpr) WalkSubtree(visit Visit) error {
 // ConvertExpr represents a call to CONVERT(expr, type)
 // CONVERT(expr USING transcoding_name) it not supported
 type ConvertExpr struct {
```

Comment:
How about going for a completely new `ConvertUsing` type ? I think that will be less confusing.